src/main/java/io/quarkus/gizmo2/desc/ConstructorDesc.java

```diff
@@ -37,7 +37,33 @@ static ConstructorDesc of(ClassDesc owner) {
     /**
      * Construct a new instance.
      *
-     * @param owner the descriptor of the class which contains the member (must not be {@code null})
+     * @param owner the descriptor of the class which contains the constructor (must not be {@code null})
+     * @param paramTypes a list of parameter types (must not be {@code null})
+     * @return the constructor descriptor (not {@code null})
+     */
+    static ConstructorDesc of(ClassDesc owner, Class<?>... paramTypes) {
+        ClassDesc[] params = new ClassDesc[paramTypes.length];
+        for (int i = 0; i < paramTypes.length; i++) {
+            params[i] = Util.classDesc(paramTypes[i]);
+        }
```

Comment:
Maybe `Stream.of(paramTypes).map(Util::classDesc).toArray(ClassDesc[]::new)`?

Reply:
Will do.

Reply:
Done.